_posts/2017-08-08-euroclojure-2017.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: EuroClojure 2017
+author: mkainula, tatut, pkalliok, viesti
+excerpt: A brief description of the presentations held in EuroClojure 2017 in Berlin.
+tags:
+- Clojure
+---
+
+EuroClojure 2017 was held in Berlin, Germany on July 20th and 21st. The venue was Kulturbrauerei, an old brewery turned into a cultural center, in the lively Prenzlauer Berg neighborhood. This year, there were seven Clojure enthusiasts from Solita attending the conference, and a couple more from our subsidiary Evolta.
+
+This post contains a quick recap of the talks and tips on which talks you should watch from Youtube (once they are posted there)
```

Comment:
lause rytmittyy väärin ikään kuin "talks and tips" kuuluisi yhteen, laittaisin esim.
of the talks, as well as tips